src/query/src/qSqlParser.c

```diff
@@ -1159,6 +1159,10 @@ void destroySqlNode(SSqlNode *pSqlNode) {
   pSqlNode->fillType = NULL;
 
   tSqlExprDestroy(pSqlNode->pHaving);
+
```

Comment:
The %destructor timestamp directive in the source code sql.y of Lemon parser generator doesn't suffice resource deallocation of timestamp tSqlExpr(s), which are stored in the external context pInfo. So, it is required to destroy the timestamp tSqlExpr(s) of time range here